src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -94,54 +103,49 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     setCurrentIndex(index);
   };
 
-  const mainClass = React.useCallback(() => {
-    switch (type) {
-      case 'borderless':
-        return 'iui-tabs-borderless';
-      case 'pill':
-        return 'iui-tabs-pill';
-      default:
-        return 'iui-tabs-horizontal';
-    }
-  }, [type]);
-
   return (
-    <>
+    <div className='iui-tabs-wrapper'>
```

Comment:
Why we need it? I thought stripe can be placed inside `iui-tabs`. Now it is container for tabs container...

Reply:
iui-tabs is a list, it should only have `li` elements inside it. I don't see the problem, the container makes sense for the content area too